catalog/resource_metastore_test.go

```diff
@@ -134,7 +134,10 @@ func TestDeleteMetastore(t *testing.T) {
 		Fixtures: []qa.HTTPFixture{
 			{
 				Method:   "DELETE",
-				Resource: "/api/2.1/unity-catalog/metastores/abc?force=false",
+				Resource: "/api/2.1/unity-catalog/metastores/abc",
+				ExpectedRequest: map[string]bool{
+					"force": false,
```

Comment:
`force=false` should not have an effect, logically...

there needs to be a PR to go SDK as well.

Reply:
SDK PR is here: https://github.com/databricks/databricks-sdk-go/pull/383. This is a draft until that is merged.